about.html

```diff
@@ -0,0 +1,129 @@
+          <li class="desk-li"><a href="#">Log in</a></li>
+      </ul>
+  </section>
+    <!-- body headline part -->
+    <section class="headline headline-about">
+      <h1>
+        "Hello!! world"
+      </h1>
+      <p class="headline__p">
+        Welcome to my DemoCapstone.
+      </p>
+  
+      <!-- box inside headline -->
+      <div class="headline__box1 box1-about">
+        Lorem, ipsum dolor sit amet consectetur adipisicing elit. Tenetur nam consequuntur libero vitae sequi aliquam, quo ad! Veritatis tempore perferendis eius fuga neque, alias possimus aliquid hic, doloribus sit illo.
+      </div>
+      <div class="headline__box2 box2-about">
+        <p class="headline__box2__p box2-about__p">
+          please contact us via email if you have any further questions about my capstone PR.<br>
+          <a href="mailto: nafeurrahmanadil@gmail.com" class="box2-about__mail">nafeurrahmanadil@gmail.com</a>
+        </p>
+      </div>
+    </section>
+  </div>
+  <section class="cc">
+    <h2 class="cc__h2">
+      Creative Commons Microverse logo
+      <span class="red-bar"></span>
+    </h2>
+    <p class="cc__p">
+      Lorem, ipsum dolor sit amet consectetur adipisicing elit. Facilis mollitia esse aut magni animi tempora.
+    </p>
+      <img src="img/mBlackBig.png" alt="" class="cc__logo">
+  </section>
+  <section class="past-cc">
+    <h2 class="past-cc__h2">
+      See the past CC Global Summits
+      <span class="red-bar"></span>
+    </h2>
+    <p class="past-cc__p">
+      Lorem ipsum, dolor sit amet consectetur adipisicing elit. Et distinctio molestiae vitae commodi, voluptatum sed.
+    </p>
+    <div class="past-cc__box">
+      <div class="past-cc__box__box1">
+        <img src="img/class2.jpg" alt="" class="past-cc__box__img">
+        <div class="past-cc__box__context">
+        <h3 class="past-cc__box__h3">2013</h3>
+        <p class="past-cc__box__p">Lorem ipsum dolor sit</p>
+      </div>
+      </div>
+      <div class="past-cc__box__box2">
+        <img src="img/class2.jpg" alt="" class="past-cc__box__img">
+        <div class="past-cc__box__context">
+        <h3 class="past-cc__box__h3">2011</h3>
+        <p class="past-cc__box__p">Lorem ipsum dolor sit</p>
+      </div>
+      </div>
+    </div>
+  </section>
+  <section class="partner">
+    <h2 class="partner__h2">
+      Partners
+      <span class="red-bar"></span>
+    </h2>
+    <div class="partner__logo">
+      <img src="img/CodeClub.svg" alt="">
+      <img src="img/NOVER.svg" alt="">
+      <img src="img/mezilla.svg" alt="">
+      <img src="img/CodeClub.svg" alt="">
+      <img src="img/mezilla.svg" alt="">
+    </div>
+  </section>
+    <footer>
+      <div class="footer-flex">
+      <div class="footer__logo">
+        <img src="img/microverse-logo.png" alt="" class="footer__logo__img">
+      </div>
+      <div class="footer__p">
+        2023 M1 Capstone project.<br>No Rights Reserved.
+      </div>
+    </div>
+    </footer>
```

Comment:
- [ ] Your footer looks good, but on the mobile version template of the about page, the colors are inverted. Kindly use the white background color with the dark text and logo to match the template.
 - Your design:
 - 
<img width="431" alt="image" src="https://user-images.githubusercontent.com/61919617/217386292-64b04d1c-2a7c-4d5b-9517-a109eb62492b.png">

 - Template:
 - 
<img width="486" alt="image" src="https://user-images.githubusercontent.com/61919617/217386416-d71985a5-0871-4ece-910f-adffe337e215.png">